tests/core/serialization/deserialize-capacity.test.ts

```diff
@@ -6,7 +6,7 @@ import { deserialize } from '../../../cocos/serialization/deserialize';
 @ccclass('Foo')
 class Foo { @property foo!: Foo | null; }
 
-const EXPECTED_CAPACITY = 900;
+const EXPECTED_CAPACITY = 750;
```

Comment:
Why change it?

Reply:
I checked with @shrinktofit 

this test case test the max depth of the  deserialize call stack, in the node version 18.17.0, the max depth cahnges to 750

Reply:
Ok.